manifest.json

```diff
@@ -125,7 +131,7 @@
           "default": false
       },
       "filename": {
-          "description": "Output filename template (%a=antenna (coil) number, %c=comments, %d=description, %e=echo number, %f=folder name, %i=ID of patient, %j=seriesInstanceUID, %k=studyInstanceUID, %m=manufacturer, %n=name of patient, %p=protocol, %s=series number, %t=time, %u=acquisition number, %v=vendor, %x=study ID; %z=sequence name). '%f' (default).",
+          "description": "Output filename template. Options: %a=antenna (coil) number, %c=comments, %d=description, %e=echo number, %f=folder name, %i=ID of patient, %j=seriesInstanceUID, %k=studyInstanceUID, %m=manufacturer, %n=name of patient, %p=protocol, %s=series number, %t=time, %u=acquisition number, %v=vendor, %x=study ID, %z=sequence name. Can combine any of the options into a phrase (e.g., '%f_%p_%t_%s'). '%f' (default).",
```

Comment:
This is more a "filename_format" (like date format "%YY-%MM-%DD").

Reply:
I agree. But we are trying to maintain as much parity with the dcm2niix tool and with former dcm2niix Gear versions.